app/views/organizations/staff/manage_fosters/edit.html.erb

```diff
@@ -1,58 +1,11 @@
-<!--page heading-->
-<header class="pt-5 pb-5">
-  <div class="container">
-    <div class="text-center">
-      <h1 class="h4">
-        <%= @application.adopter_foster_account.user.first_name %>
-        <%= @application.adopter_foster_account.user.last_name %>'s application for
-        <%= @application.pet.name %>
-      </h1>
-    </div>
-  </div> <!--container-->
-</header>
+<% breadcrumb :edit_foster, @foster %>
 
-<!--main section-->
-<section class="pb-5" id="about_us">
-  <div class="container">
-    <div class="row">
-      <div class="col-md-6 mx-auto card p-5 ">
-        <div class='d-flex flex-row-reverse'>
-          <%= link_to "#{@application.adopter_foster_account.user.first_name}'s Profile",
-              staff_profile_review_path(@application.adopter_foster_account.adopter_foster_profile.id),
-              class: 'text-decoration-none bigger' %>
-        </div>
+<%= render "components/dashboard/page" do |p| %>
+  <% p.header_title do %>
+    Edit Foster
```

Comment:
Currently, editing is done with the turbo frames. However, I included this edit page for cases where JavaScript is disabled.